integration/sbom_test.go

```diff
@@ -57,6 +58,16 @@ func TestSBOM(t *testing.T) {
 			},
 			golden: "testdata/fluentd-multiple-lockfiles.json.golden",
 		},
+		{
+			name: "scan SBOM into SBOM",
+			args: args{
+				input:        "testdata/fixtures/sbom/pywin32-cyclonedx.json",
```

Comment:
Can we re-use the existing json, `fluentd-multiple-lockfiles-cyclonedx.json`? You can change this file to use UUID in BOM-Ref.

Reply:
the testcase is updated `fluentd-multiple-lockfiles-cyclonedx.json`. 
I wanted to use `fluentd-multiple-lockfiles.cdx.json.golden` also, but this cdx result is a large, because it's result for `fluentd-multiple-lockfiles.tar.gz`. `fluentd-multiple-lockfiles-cyclonedx.json` is shorter. 
so I've created a new golden file.